### PR TITLE
Create DungeonHelper section and move relative stuff

### PR DESCRIPTION
**Pull request by Bigsie**

### CODE REVIEW

File: System/Functions/Custom.lua [hunk 1]
```diff
@@ -1035,3 +1035,7 @@ function BWCheck()
         end
     end
 end
+
+function getCurrentZoneId()
+    return select(2, GetMapId())
```
Comment (kuukuukuatchu) [inline System/Functions/Custom.lua:1040]: What is this?  I can't find GetMapId() anywhere in WoW API

File: System/Misc/Zu/BossManager.lua [hunk 2]
```diff
@@ -1,17 +1,20 @@
...
-                local x2, y2, z2 = ObjectPosition(object)
-                local distance = math.sqrt(((x2 - x1) ^ 2) + ((y2 - y1) ^ 2) + ((z2 - z1) ^ 2))
-                if ID == 130099 and distance < 10 then
-                    InteractUnit(object)
+    local function clickHelper(ID)
+        for _,v in pairs(br.player.enemies.get(40,nil,true)) do
+            if ID == getUnitID(v) then
+                local distance = GetDistanceBetweenObjects("player", v)
+                if distance < 10 then
+                    InteractUnit(v)
```
Comment (kuukuukuatchu) [inline System/Misc/Zu/BossManager.lua:7]: There should be some kind of delay here so it's not instant and bot like

File: System/Functions/Custom.lua (hunk 1 above)
Comment (Bigsie) [reply System/Functions/Custom.lua:1040]: https://ewtwow.com/EWT/ewt.lua

File: System/Misc/Zu/BossManager.lua (hunk 2 above)
Comment (Bigsie) [reply System/Misc/Zu/BossManager.lua:7]: Roger, ill add some

Comment (Bigsie) [reply System/Misc/Zu/BossManager.lua:7]: Cant implement such a delay :/

**Pull request merged**